### DISCUSSION TITLE
2 column note lists

### DISCUSSION THREAD

**Discussion by beatnutz** [standardnotes/forum#2755] [Suggestions]
Hi and thank you for this fantastic app.

I would love to have the option (either in the settings or icon in the corner) to be able to split the list of notes in half in to two columns.

On mobile it's a bit of an issue to see so few notes before having to scroll. Things get buried fast.

Thank you

**Comment by JaspalSuri** [comment]
Hi there, while this feature currently isn't planned, you can use the option to hide your notes' previews in order to condense the notes list and maximize the amount of notes that simultaneously appear on your device's display. Please open the left sidebar, tap on you username or the gear icon, then tap on **Hide note previews**. I hope this helps! 🙂
